src/loader.rs

```diff
@@ -16,6 +16,34 @@ impl Loader {
       path: path.to_owned(),
       io_error,
     })?;
-    Ok(self.arena.alloc(src))
+
+    // Parse include directives.
+    // Include directives are lines that look like:
+    // #include "relative/path/to/file"
+    let include_regexp = Regex::new(r#"^#include "([^"]+)"$"#).unwrap();
```

Comment:
Your PR description uses the syntax "@include". Here you use "#include", also in the comment. Let's unify.

Reply:
Typo from the implementation we used at my work which was `#include`. I'll fix it up here in a minute.